src/Controller/ArticlesController.php

```diff
@@ -978,7 +978,7 @@ private function articlePageArguments(Request $request, string $id, int $version
                 $infoBars = [];
                 if (false === $isMagazine) {
                     $infoBarText = sprintf(
-                        'eLife\'s peer-review process is changing. From early next year, we will no longer make accept/reject decisions after peer review. <a href="%s" class="">About the new process.</a>',
+                        'eLife\'s peer-review process is changing. In the new process, we will no longer make accept/reject decisions after peer review. <a href="%s">Find out more.</a>',
```

Comment:
Good work. 

One comment about the link. When a user uses a screen reader they tend to tab through the links and they will come across this link and it will say "Find out more".

Find out more of what? They will not know where as the previous text before the change in the link gave a description of where the link is going to.

I would look at changing this